src/utils/getApplicableFormats.ts

```diff
@@ -3,6 +3,12 @@ import { RegularNode, SchemaNodeKind } from '@stoplight/json-schema-tree';
 import { COMMON_JSON_SCHEMA_AND_OAS_FORMATS } from '../consts';
 
 export function getApplicableFormats(schemaNode: RegularNode): [type: SchemaNodeKind, format: string] | null {
+  // Format is handled for contentMediaType type i.e. binary format
+  if (schemaNode.types !== null && schemaNode.fragment['contentMediaType'] === 'application/octet-stream') {
+    for (const type of schemaNode.types) {
```

Comment:
why a loop here if its just going to return?

Reply:
Updated the file. pls review once.